sympy/combinatorics/fp_groups.py

```diff
@@ -494,6 +496,97 @@ def elements(self):
         P, T = self._to_perm_group()
         return T.invert(P._elements)
 
+    @property
+    def is_polycyclic(self):
+        if self.is_solvable:
+            return True
+        return None
+
+    def is_subgroup(self, other):
+        """
+        Return ``True`` if all elements of ``self`` belong to ``other``.
+        """
+        _grp1 = self._to_perm_group()[0]
+        _other = self._to_perm_group()[0]
+        return _grp1.is_subgroup(_other, strict=False)
+
+    def compute_polycyclic_series(self):
+        '''
+        ToDo - Update docstring.
+        '''
+        if not self.is_polycyclic:
+            raise NotImplementedError("Polycyclic methods are only"
+                                        "implemented for FpGroups")
+        # Compute the derived series.
+        derived_series = self.derived_series()
+        derived_series.insert(0, self)
```

Comment:
> so it would make more sense to say derived_series = [self.generators] + self.derived_series()

Wouldn't it be better to leave a few groups as `FpGroup` in the subnormal series wherever possible? Because that way, we could avoid the computation of presentation at least for some groups when we try to find the relative orders of the polycyclic group.

Reply:
I mean, the original group will have a trivial coset table so probably wouldn't take long anyway, but I see your point. Could you perhaps find all the necessary subgroup presentations here, before adding the original group to them?